### PR TITLE
Initial WIP implementation of sync using http and T ##sync

### PR DESCRIPTION
**Pull request by radare**

### CODE REVIEW

**Comment by radare** [comment]
There's a lot more work to be done, but that's just a PoC, and fits within r2

**Comment by XVilka** [comment]
For the reference - this was the original issue: https://github.com/radare/radare2/issues/7410

File: libr/core/cmd_log.c [hunk 1]
```diff
@@ -180,15 +250,52 @@ static int cmd_log(void *data, const char *input) {
...
 		break;
-	case 'T':
+	case 'T':  // Ts ? as ms?
 		if (r_config_get_i (core->config, "scr.interactive")) {
 			textlog_chat (core);
 		} else {
 			eprintf ("Only available when the screen is interactive\n");
 		}
 		break;
+	case '=':
```
Comment (ret2libc) [inline libr/core/cmd_log.c]: please add `// T=` just to help grepping it

File: libr/core/cmd_log.c [hunk 2]
```diff
@@ -180,15 +250,52 @@ static int cmd_log(void *data, const char *input) {
...
+				}
+				r_cons_break_pop ();
+			} else {
+				// TODO: Sucks that we can't enqueue functions, only commands
+				eprintf ("Background thread syncing with http.sync started.\n");
+				RCoreTask *task = r_core_task_new (core, true, "T=&&", NULL, core);
+				r_core_task_enqueue (core, task);
+			}
+		} else {
+			if (atoi (input + 1) > 0 || (input[1] == '0')) {
```
Comment (ret2libc) [inline libr/core/cmd_log.c]: how is `T=` supposed to work? Does it expect a number as parameter? It's not clear from the help msg.

**Comment by radare** [comment]
this pr can become quite big.. because it dpeends other features and changes.. so i dont wanna make it much bigger. so pls review this partial impl and ill continue with separate PRs

File: libr/core/cmd_log.c (hunk 2 above)
Comment (radare) [reply libr/core/cmd_log.c]: yes, it expects a number but it's optional, in that case it pulls from the last id retrieved.

**Pull request merged**